packages/@aws-cdk/aws-rds/lib/cluster-parameter-group-ref.ts

```diff
@@ -0,0 +1,60 @@
+import cdk = require('@aws-cdk/cdk');
+import { DBClusterParameterGroupName } from './rds.generated';
+
+/**
+ * A cluster parameter group
+ */
+export abstract class ClusterParameterGroupRef extends cdk.Construct {
+    /**
+     * Import a parameter group
+     */
+    public static import(parent: cdk.Construct, id: string, props: ClusterParameterGroupRefProps): ClusterParameterGroupRef {
+        return new ImportedClusterParameterGroup(parent, id, props);
+    }
+
+    /**
+     * Name of this parameter group
+     */
+    public abstract readonly parameterGroupName: DBClusterParameterGroupName;
+
+    /**
+     * Export this parameter group
+     */
+    public export(): ClusterParameterGroupRefProps {
+        return {
+            parameterGroupName: new DBClusterParameterGroupName(
+                    new cdk.Output(this, 'ParameterGroupName', { value: this.parameterGroupName }).makeImportValue())
+        };
+    }
+
+    /**
+     * Set a parameter on this parameter group
+     */
+    public abstract setParameter(key: string, value: string | undefined): void;
```

Comment:
Why is this API on the `Ref` and not on the concrete object? Does it make sense to allow passing in a Ref without the ability to modify the parameters?

Reply:
Good point, my API design originally had an automatically-created parameter group and a helper `setParameter()` on the Cluster class, but that didn't pan out because:

* Can't really guess at `family`, there are ~2 million options to choose from.
* We cannot create an empty parameter group through CloudFormation, so we'd have to guess at a no-op parameter value to set, which will be different per family.

Ultimately I punted on that change (even though the experience would be better). So now this doesn't make sense anymore either, agreed.